### PR TITLE
Add thumbnail support

### PR DESCRIPTION
**Pull request by ethanwharris**
# Before submitting

- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

## What does this PR do?

Fixes # (issue).

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Review by SeanNaren — APPROVED** [review]

**Review by tchaton — APPROVED** [review]

File: .actions/helpers.py [hunk 1]
```diff
@@ -349,37 +349,77 @@ def _get_card_item_cell(path_ipynb: str) -> Dict[str, Any]:
...
+            path_meta = path_ipynb.replace(".ipynb", ".yaml")
+            path_thumb = HelperCLI._resolve_path_thumb(path_ipynb, path_meta)
+
+            if path_thumb is not None:
+                new_thumb = os.path.join(docs_root, path_docs_images, path_thumb)
+                old_path_thumb = os.path.join(path_root, HelperCLI.DIR_NOTEBOOKS, path_thumb)
+                os.makedirs(os.path.dirname(new_thumb), exist_ok=True)
+
+                print(f"{old_path_thumb} -> {new_thumb}")
+
```
Comment (Borda) [inline .actions/helpers.py:413]:
```suggestion
```

File: .actions/ipynb-render.sh [hunk 2]
```diff
@@ -40,6 +49,11 @@ else
 fi
 
 cp $meta_file $pub_meta_file
-git add ".notebooks/$1.yaml"
+git add $pub_meta_file
+
+if [ ! -z $thumb_file ]; then
+  cp $thumb_file $pub_thumb_file
```
Comment (Borda) [inline .actions/ipynb-render.sh:55]: isn't the copy already in the CLI with `copyfile(old_path_thumb, new_thumb)`

File: .github/workflows/ci_docs.yml [hunk 3]
```diff
@@ -63,13 +63,15 @@ jobs:
...
               meta=( $(ls "$line"/.meta.*) )
+              thumb=( $(ls "$line"/.thumb.* 2>/dev/null || echo "") ) || true
               # create folder of it is missing
               pub_dir=$(dirname ".notebooks/$line")
               printf "Dir: $pub_dir"
               mkdir -p $pub_dir
               # simulate render as somple move
               mv $ipynb ".notebooks/$line.ipynb"
               mv $meta ".notebooks/$line.yaml"
+              [ ! -z $thumb ] && mv $thumb .notebooks/${thumb/"$line"\/.thumb/$line} || true
```
Comment (Borda) [inline .github/workflows/ci_docs.yml:74]: move? why not copy?

File: .actions/ipynb-render.sh (hunk 2 above)
Comment (ethanwharris) [reply .actions/ipynb-render.sh:55]: This copies from original location to `.notebooks`, then CLI copies from `.notebooks` to `docs/source/_static/images`

File: .github/workflows/ci_docs.yml (hunk 3 above)
Comment (ethanwharris) [reply .github/workflows/ci_docs.yml]: Just copied it from the above lines. Changed :smiley:

**Review by Borda — APPROVED** [review]

**Pull request merged**